CHANGELOG

```diff
@@ -1,3 +1,11 @@
+2016-03-04: v0.5.2
+  Add EnforceRFC952 to sample config
+  Improve performance of record generation
```

Comment:
Do you want to do that before we proceed? I primarily did this PR to get the build for testing. We can stall if you don't feel comfortable